packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -231,10 +234,10 @@ class FleatherEditor extends StatefulWidget {
 
```

Comment:
[nitpick] I see you change to `State<FleatherFIeld>` the return type in `FleatherField`
Can we align & change to `State<FleatherEditor>`

Reply:
Done. @amantoux